src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callsCount = 1;
+  const getSum = (left) => {
+    return (right) => {
```

Comment:
```suggestion
  const getSum = (x) => {
    return (y) => {
```
не совсем понятно, что значит left и right